components/dockmanager/overview.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Overview
+page_title: DockManager Overview
+description: Overview of the DockManager for Blazor.
+slug: dockmanager-overview
+tags: telerik,blazor,dockmanager,overview
+published: True
+position: 0
+---
+
+# Blazor DockManager Overview
+
+The <a href="https://www.telerik.com/blazor-ui/dock-manager" target="_blank">Blazor DockManager component</a> is a versatile tool that enables users to manage and organize multiple panes within a single container. It supports features like docking, undocking, resizing, and repositioning, offering a flexible and customizable layout.
+
+
+## Creating Blazor DockManager
+
+1. Add the `TelerikDockManager` tag.
+2. Use `<DockManagerPanes>` to structure the main docked layout.
+3. Within <DockManagerPanes>, add:
+    * `<DockManagerContentPane>` for standalone panes.
+    * `<DockManagerSplitPane>` to create resizable sections containing multiple panes.
+    * `<DockManagerTabGroupPane>` to enable tabbed panes.
+4. Define `HeaderTemplate` inside each pane to set the headers text.
+5. Populate the `Content` section of each pane with the desired UI elements.
```

Comment:
If you use tags, then use tags for the `HeaderTemplate` and `Content` too. Otherwise it may look like these are used differently.